### PR TITLE
Fix certificate renewal

### PR DESCRIPTION
**Pull request by varks**
Update secrets in remote enforcers in order to update the token engine.

### CODE REVIEW

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "apotests",
    "pr-id": "1283",
    "commit-sha": "48185f71afb2c7f3076ae399e55a83c9f4dc7aa8"
  },
  {
    "component": "trireme-lib",
    "pr-id": "512",
    "commit-sha": "06bafc9804e0aabd71237328de26f3fdf0ae5684"
  },
  {
    "component": "enforcerd",
    "pr-id": "566",
    "commit-sha": "673d7fe3e821c9cd33a2806ac9a2808e1dbf6ca3"
  },
  {
    "component": "squall",
    "pr-id": "341",
    "commit-sha": "a0ec7c59be930140f0687517f44105d29fd656b5"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "apotests",
    "pr-id": "1283",
    "commit-sha": "48185f71afb2c7f3076ae399e55a83c9f4dc7aa8"
  },
  {
    "component": "trireme-lib",
    "pr-id": "512",
    "commit-sha": "c259b62f06c9316b0cdee358b7ef00c3d8895db6"
  },
  {
    "component": "enforcerd",
    "pr-id": "566",
    "commit-sha": "673d7fe3e821c9cd33a2806ac9a2808e1dbf6ca3"
  },
  {
    "component": "squall",
    "pr-id": "341",
    "commit-sha": "034cae23fca095bf238f3bf399d01215bbc54417"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "squall",
    "pr-id": "341",
    "commit-sha": "034cae23fca095bf238f3bf399d01215bbc54417"
  },
  {
    "component": "apotests",
    "pr-id": "1283",
    "commit-sha": "48185f71afb2c7f3076ae399e55a83c9f4dc7aa8"
  },
  {
    "component": "trireme-lib",
    "pr-id": "512",
    "commit-sha": "b30f493e4437fcfa722d327316eeec6eadf1cab8"
  },
  {
    "component": "enforcerd",
    "pr-id": "566",
    "commit-sha": "3806a7d5f9aae9f225d15f549ffe63a76a55c5e5"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "trireme-lib",
    "pr-id": "512",
    "commit-sha": "b30f493e4437fcfa722d327316eeec6eadf1cab8"
  },
  {
    "component": "enforcerd",
    "pr-id": "566",
    "commit-sha": "3806a7d5f9aae9f225d15f549ffe63a76a55c5e5"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "trireme-lib",
    "pr-id": "512",
    "commit-sha": "a04edd2bc8f900380f7a7498571f2c846ea0e668"
  },
  {
    "component": "enforcerd",
    "pr-id": "566",
    "commit-sha": "3806a7d5f9aae9f225d15f549ffe63a76a55c5e5"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "trireme-lib",
    "pr-id": "512",
    "commit-sha": "a04edd2bc8f900380f7a7498571f2c846ea0e668"
  },
  {
    "component": "enforcerd",
    "pr-id": "566",
    "commit-sha": "12140e28d94d3972bfcd541ae6fbf3553eb6ba9d"
  }
]
```

File: controller/pkg/remoteenforcer/remoteenforcer_linux.go [hunk 1]
```diff
@@ -368,8 +368,9 @@ func (s *RemoteEnforcer) UpdateSecrets(req rpcwrapper.Request, resp *rpcwrapper.
 
 	cmdLock.Lock()
 	defer cmdLock.Unlock()
-	if s.enforcer != nil {
-		return nil
+	if s.enforcer == nil {
+		zap.L().Error(resp.Status)
```
Comment (amitlimaye) [inline controller/pkg/remoteenforcer/remoteenforcer_linux.go]: Don't need this error message

**Review by amitlimaye — APPROVED** [review]

**Pull request merged**